meetballs/urls.py

```diff
@@ -26,4 +26,7 @@
     path('game-events/', include("game_event.urls")),
     path('login/', app_views.loginUser, name='loginUser'),
     path('logout/', app_views.logoutUser, name='logoutUser'),
+    path('profile/<id>/', app_views.profile, name='profile'),
+    path('profile/<id>/edit/', app_views.edit_profile, name='edit profile'),
```

Comment:
I believe this could potentially lead to a security vulnerability. Instead of extracting the ID from the URL, retrieving the ID from the client request is recommended. Doing so prevents unauthorized access to modify profiles by simply visiting a specific URL such as http://127.0.0.1:8000/profile/2/edit/.

Reply:
correct, but how are we suppose to implement it if we want to allow other users to view each others profile pages ?

Reply:
You should use `@login_required` decorator, which ensures that only logged-in users can access this page. Next, you can use request.user.id to retrieve the relevant user ID within the views. You can then perform your logic using that ID.

In line 29, it's okay to leave it as it is if you want to make the profile public, allowing everyone to access it. However, you need to ensure that if `request.user.id != id_from_url`, the edit profile button will be disabled